app/Models/GamesAward.php

```diff
@@ -56,19 +56,19 @@ class GamesAward extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->